app/code/Magento/CatalogSearch/Ui/DataProvider/Product/AddFulltextFilterToCollection.php

```diff
@@ -38,13 +38,30 @@ public function addFilter(Collection $collection, $field, $condition = null)
     {
         /** @var $collection \Magento\Catalog\Model\ResourceModel\Product\Collection */
         if (isset($condition['fulltext']) && (string)$condition['fulltext'] !== '') {
-            $this->searchCollection->addBackendSearchFilter($condition['fulltext']);
-            $productIds = $this->searchCollection->load()->getAllIds();
+            $this->backendCollection->addSearchFilter((string) $condition['fulltext']);
+            $productIds = $this->backendCollection->load()->getAllIds();
+            $simpleProductIds = [];
             if (empty($productIds)) {
                 //add dummy id to prevent returning full unfiltered collection
                 $productIds = -1;
+            } else {
+                $simpleProductIds = $this->getSimpleProductIds($this->backendCollection->getItems());
             }
-            $collection->addIdFilter($productIds);
+            $collection->addIdFilter(array_merge($simpleProductIds, $productIds));
         }
     }
+
+    /**
+     * @param array $products
+     * @return array
+     */
+    private function getSimpleProductIds(array $products): array
```

Comment:
The context with this additional query is that elasticsearch does not store the simple products data from the configurable. This solution is not perfect but it does mean the backend sees the simple products as well as the parents